config/config.localnet.toml

```diff
@@ -29,6 +29,7 @@ strict_genesis_parsing = false
 [cosmosapp]
 http_rpc_url = "http://127.0.0.1:1317"
 insecure = false
+v0_43_0 = 0
```

Comment:
Some suggestion:

can be more descriptive (e.g. `enable_height_v0_43_0`) and maybe move to `blockchain` section

or a separate section
```
[version_enabled_height]
v0_43_0
```

Reply:
Good suggestion!